external/json/cJSON.c

```diff
@@ -721,43 +721,42 @@ static cJSON_bool parse_string(cJSON * const item, parse_buffer * const input_bu
                 goto fail;
             }
 
-            switch (input_pointer[1])
-            {
-                case 'b':
-                    *output_pointer++ = '\b';
-                    break;
-                case 'f':
-                    *output_pointer++ = '\f';
-                    break;
-                case 'n':
-                    *output_pointer++ = '\n';
-                    break;
-                case 'r':
-                    *output_pointer++ = '\r';
-                    break;
-                case 't':
-                    *output_pointer++ = '\t';
-                    break;
-                case '\"':
-                case '\\':
-                case '/':
-                    *output_pointer++ = input_pointer[1];
-                    break;
+            switch (input_pointer[1]) {
+            case 'b':
+            	*output_pointer++ = '\b';
```

Comment:
make sure to preserve style,

cleanup change are preffered is separated changes